src/test/unit/requests/subscriptionRequest.test.ts

```diff
@@ -2,103 +2,88 @@ import { SubscriptionRequests } from '../../../main/resources/requests/subscript
 import sinon from 'sinon';
 import fs from 'fs';
 import path from 'path';
-import { dataManagementApi, subscriptionManagementApi } from '../../../main/resources/requests/utils/axiosConfig';
+
+import { subscriptionManagementApi } from '../../../main/resources/requests/utils/axiosConfig';
+import {Subscription} from '../../../main/models/subscription';
 
 const userIdWithSubscriptions = 1;
 const userIdWithoutSubscriptions = 2;
 const nonExistingUserId = 777;
-const validUrn = '123456789';
-const invalidUrn = '1234';
+
 const subscriptionActions = new SubscriptionRequests();
 const mockedCaseSubscription = {
   name: 'Wyman Inc Dispute',
-  reference: 'T20217010',
-  dateAdded: '1632351600',
+  reference: 'T485913',
 };
 const mockedCourtSubscription = {
-  name: 'Mutsu Court',
-  dateAdded: '1632351600',
+  name: 'Court 1',
+  dateAdded: '2022-01-14T11:42:57.847708',
 };
 
 const errorResponse = {
   response: {
     data: 'test error',
   },
 };
-const errorMessage = {
-  message: 'test',
-};
 const errorRequest = {
   request: 'test error',
 };
+const errorMessage = {
+  message: 'test',
+};
 
-const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/subscriptionListResult.json'), 'utf-8');
-const subscriptionsData = JSON.parse(rawData);
-const stub = sinon.stub(dataManagementApi, 'get');
+const rawData2 = fs.readFileSync(path.resolve(__dirname, '../../../test/unit/mocks/userSubscriptions.json'), 'utf-8');
+const subscriptionsData2 = JSON.parse(rawData2);
+const stub = sinon.stub(subscriptionManagementApi, 'get');
 const subscriptionManagementStub = sinon.stub(subscriptionManagementApi, 'post');
 
 describe(`getUserSubscriptions(${userIdWithSubscriptions}) with valid user id`, () => {
-  const userSubscriptions = subscriptionActions.getUserSubscriptions(userIdWithSubscriptions);
-  it('should return user subscription object', () => {
-    expect(userSubscriptions.courtSubscriptions.length).toBeGreaterThan(0);
-    expect(userSubscriptions.caseSubscriptions.length).toBeGreaterThan(0);
-  });
+  stub.withArgs(`/subscription/user/${userIdWithSubscriptions}`).resolves(subscriptionsData2);
 
-  it('should have mocked object in the case subscriptions list', () => {
-    expect(userSubscriptions.caseSubscriptions.filter((subscription) =>
-      subscription.reference === mockedCaseSubscription.reference).length).toBe(1);
-  });
+  it('should return user subscription object', async () => {
+    const userSubscriptions = await subscriptionActions.getUserSubscriptions(userIdWithSubscriptions) as Subscription;
```

Comment:
why are you setting a return type here as `Subscription` and not doing it elsewhere? For the consistency either remove it or put it everywhere (lines 53, 60). 
It also doesn't read well, call is fetching `user subscriptions` and return type is `Subscription`? Should the model be renamed to `UserSubscriptions` as it contains arrays of `case and court subscriptions`?

Reply:
removed return type and renamed model